### ISSUE TITLE
Thunderhub v0.13.7 Install Failure

### ISSUE THREAD

**Issue by tehelsper** [apotdevin/thunderhub#431]
**Describe the problem/bug**
Install steps fail now. Worked earlier on same version.

**Your environment**
Installing via "npm install"

**To Reproduce**
Install Thunderhub

**Expected behavior**
I expect Thunderhub to install properly.

**Actual behavior**
I get the following error:

```
+ sudo -u bitcoin npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-qr-reader@2.2.1
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!   peerOptional react@"^16.8.0 || ^17.0.0" from @apollo/client@3.5.9
npm ERR!   node_modules/@apollo/client
npm ERR!     @apollo/client@"^3.5.9" from the root project
npm ERR!   30 more (@emotion/react, @visx/axis, @visx/bounds, @visx/chord, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"~16" from react-qr-reader@2.2.1
npm ERR! node_modules/react-qr-reader
npm ERR!   react-qr-reader@"^2.2.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@16.14.0
npm ERR! node_modules/react
npm ERR!   peer react@"~16" from react-qr-reader@2.2.1
npm ERR!   node_modules/react-qr-reader
npm ERR!     react-qr-reader@"^2.2.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/bitcoin/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bitcoin/.npm/_logs/2022-04-04T04_07_38_457Z-debug-0.log
```

Edit: Adding "--legacy-peer-deps" resolves the install issue.

**Comment by apotdevin** [comment]
Hey, which version is this?

**Comment by tehelsper** [comment]
It was in 13.7. Seems to be resolved in v13.10.
